themes/default/templates/suspend.html

```diff
@@ -1,4 +1,7 @@
-<h1 data-localise="suspend.paused header"></h1>
+<h1 >
+    <span data-bind="visible: infoPage()=='suspend'" data-localise="suspend.paused header" ></span>
+    <span data-bind="visible: infoPage()=='resumed'" data-localise="suspend.resumed header" ></span>
```

Comment:
You'll need to add a "suspend.resumed header" key to `locales/en-GB.js`, containing the string "Attempt resumed"